NikoBracket/HomeViewController.m

```diff
@@ -12,6 +12,15 @@
 #import "HomeViewCell.h"
 #import "UIImageView+AFNetworking.h"
 
+static NSMutableArray *teamsInTournament;
+static int numOfPastGames;
+static int kRound1 = 0;
+static int kRound2 = 1;
+static int kRound3 = 2;
+static int kRound4 = 3;
+static int kSemiFinal = 4;
+static int kFinal = 5;
```

Comment:
are these the same as from the create bracket controller? maybe it does make sense to have a constants file for these.